app/controllers/EmailController.java

```diff
@@ -19,13 +20,17 @@
 @Controller("emailController")
 public class EmailController extends BaseController {
 
+    private BridgeConfig bridgeConfig;
```

Comment:
I'm actually more concerned that, as written, EmailController.bridgeConfig will have the correct value, but the parent BaseController.bridgeConfig will be null.

This is unlikely to cause a crash, since BaseController.bridgeConfig is only used in getStudyIdentifier(), which is never called by EmailController. But if in the future, someone adds code in BaseController that uses bridgeConfig, it may cause an obscure crash.

I strongly recommend changing it to getBridgeConfig(). It's an easy fix to do now, and it'll save us some hassle in the future.